projects/elife.yaml

```diff
@@ -2169,3 +2169,27 @@ microsimulation-demo:
         ports:
             1280: 80
             1281: 8081
+
+bioprotocol:
+    repo: https://github.com/elifesciences/bioprotocol-service
```

Comment:
no formula mentioned? This influences some `builder` behavior, I think

Reply:
I hadn't gotten to that, I'll do it today.

Reply:
done